sourcegraph/sourcegraph
# Code Insights: Improve dashboard card Voice Over navigation

Part of https://github.com/sourcegraph/sourcegraph/issues/41680

## Background
In this PR, we add list roles for the grid (the original issue discussion on why we decided to use list roles instead of grid roles  https://github.com/sourcegraph/sourcegraph/issues/41680). Keyboard navigation over the dashboard card will be addressed in a further PR. 

Also I had to update `react-grid-layout` package to be able to get reference of the grid element. 

## Test plan
- Open the Code Insights dashboard page
- Turn on Voice over 
- Try to go through dashboard insight cards
- You should hear list-item announcements like "Insights card, 1 of 5"



## App preview:

- [Web](https://sg-web-vk-improve-dashboard-keyboard.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
